setup.py

```diff
@@ -85,6 +85,7 @@ def build_extension(self, ext):
         "numpy",
         "pymbar>3.0.4,<4",
         "pyyaml",
+        "rdkit",
```

Comment:
Good call -- I think this really should've been here before too. Do you think we should also add `OpenEye-toolkits` and `OpenMM` while we're at it? (not that they would be installed with pip; this would just be an assertion that those packages are available for the timemachine installation to succeed)